docs/api-reference/experimental/README.md

```diff
@@ -0,0 +1,3 @@
+# Caveat Emptor!
```

Comment:
I recommend keeping the title of the `README.md` to the name of the module, add the latin bewares below.

If not you are using ocular-gatsby to build these the TOC entry will be auto extracted to `Caveat Emptor!` which will likely look strange.

Reply:
That is intended. I'm ok with the docs having some personality, and I want the status of experimental modules to be clear. I'll change the name of the file, though, since this isn't exactly a "README" (and none of the other modules have on).